tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -44,26 +42,27 @@ class TelemetryValidator(object):
 
   def __init__(self, entities, timeout, callback):
     super().__init__()
-    self.entities = dict(filter((lambda e: e[1].translation),
+    self.entities = dict(filter((lambda entities_with_translations:
```

Comment:
this is backward.  The output is entities with translations.  The filter is taking entities and removing those without translations.